CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # [vNext]
+* Fixed a mispelling in an error message in the DefineStepsCommand.
```

Comment:
There should between the header and the content.

Reply:
I would not even list it in the changelog....

Reply:
I've submitted a commit which takes the comment out.